packages/datadog-plugin-fetch/test/integration-test/server.mjs

```diff
@@ -0,0 +1,16 @@
+import 'dd-trace/init.js'
+import http from 'http'
+import express from 'express'
+
+const app = express()
+
+app.get('/user', (req, res) => {
+  res.status(200).send()
+})
+const server = http.createServer(app)
+
+server.listen(0, 'localhost', () => {
+  const port = server.address().port
+  global.fetch(`http://localhost:${port}/user`)
+  process.send({ port })
```

Comment:
It looks like the app is making two http servers? What port is express listening on? It kinda seems like this request handler might be making a request to itself. If you put in a `console.log` do you see multiple lines get logged?

Reply:
Oh i think i forgot to go back and update this file